### PR TITLE
Discard master-slave terminology

### PR DESCRIPTION
**Pull request by bopopescu**
For diversity reasons, it would be nice to try to avoid 'master' and 'slave' terminology in this repository which can be associated to slavery. The master-slave terminology could be problematic for people in several countries which has the history of slavery like Romania, USA and many others. Thank you for considering the proposal. Let me know if any changes in the PR are needed, I would be happy to implement them.

### CODE REVIEW

**Comment by TerrorBite** [comment]
Hi, @bopopescu! Thanks for your pull request. I do agree with your sentiment, and this is something I've actually taken action on myself — if you would observe my project https://github.com/terrorbite/textgame you will find that there is only one branch named `main`.

However, I am not sure it's even worth keeping the Mediasnak project up, let alone merging new changes in, for the following reasons:

* This was developed as a university project back in 2012, and hasn't been updated since. It served its purpose when my team and I passed the subject.
* It's written in Python 2.4, which is woefully outdated. Google App Engine doesn't even support that version of Python any more.
* All of your suggested changes are in code from the Django and Boto libraries, rather than in my own code. Have you considered making a pull request directly to those libraries?

In short, I'll probably just pull this code offline anyway. Before I do though, what are your thoughts?

**Comment by odiferousmint** [comment]
Mate, he has posted the same crap all over GitHub, currently he is at the 3171st project. He is going to be reported as a spammer.
2020-10-15-192542_866x698_scrot

Look at his profile and his pull requests. He is just mass-shitposting over GitHub.
